src/components/ADempiere/LockRecord/index.vue

```diff
@@ -0,0 +1,169 @@
+<!--
+ ADempiere-Vue (Frontend) for ADempiere ERP & CRM Smart Business Solution
+ Copyright (C) 2017-Present E.R.P. Consultores y Asociados, C.A.
+ Contributor(s): Edwin Betancourt EdwinBetanc0urt@outlook.com www.erpya.com
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
```

Comment:
Please put it in the path: src/components/ADempiere/ContainerOptions/LockRecord

Reply:
The path has been changed to `src/components/ADempiere/ContainerOptions/LockRecord`